front/style/style.css

```diff
@@ -0,0 +1,113 @@
+.dflex {
+  display: flex;
+}
+.redBoxSh {
+  box-shadow: 0 0 5px 1px #ff00008c;
+}
+.wrapWrap {
+  flex-wrap: wrap;
+}
+.dIgrid {
+  display: inline-grid;
+}
+.dGrid {
+  display: grid;
+}
+input {
+  outline: none;
+}
+body {
+  background-color: #7e8f9a;
+}
+h1 {
+  text-align: center;
+  font-weight: 100;
+  font-size: 3.5rem;
+}
+#wrapper {
+  margin: 0 auto;
+  max-width: 1200px;
+  background-color: #657580;
+}
+#calculations {
+  width: calc(100% - 5.4rem);
+  margin: 3.5rem auto 0 auto;
+  grid-template-columns: 48% 27.5% 25%;
+  grid-gap: 0 1rem;
+}
+button {
+  font-size: 1.4rem;
+}
+.fields span {
+  margin-bottom: 0.5rem;
+  font-size: 1.44rem;
+  white-space: nowrap;
+}
+.fields input {
+  border-radius: 5px;
+  outline: none;
+  border: 0;
+  width: 100%;
+  max-width: 29.5rem;
+}
+.fields button,
+.fields input {
```

Comment:
Why do you write "fields" twice?

Reply:
It is simple selector. Pay your attention for that moment in the picture is double selectors. This better then 
` field button{
  line-height:1.5; // f.e.x.
}
field input{
  line-height:1.5;
}` 
If u want that I use classes and IDs everywere dont forget it does negative influence for searching machine.